components/script/layout_dom/node.rs

```diff
@@ -97,7 +97,9 @@ impl<'dom, LayoutDataType: LayoutDataTrait> ServoLayoutNode<'dom, LayoutDataType
             phantom: PhantomData,
         }
     }
-
+    /// # Safety
+    ///
+    /// This function should not be called if there is no TrustedNodeAddress.
```

Comment:
I don't think this is exactly right. Can you remove the addition of the `Safety` sections from this change? These need to be written carefully after studying the functions that they decorate.

Reply:
Okay sir